etl/steps/data/garden/emissions/2023-05-02/national_contributions.excluded_countries.json

```diff
@@ -0,0 +1,9 @@
+[
+    "ANNEXI",
+    "ANNEXII",
+    "BASIC",
+    "EIT",
+    "LMDC",
+    "NONANNEX",
+    "Pacific Islands (Palau)"
```

Comment:
Is this not equivalent to the country Palau?

Reply:
They have data for both "Pacific Islands (Palau)" and "Palau". In GCP dataset this also happened, but it turned out that the former had data only until a certain year, and from then on, the latter had data. But now we had data for both countries overlapping. I could have looked deeper into this to figure out a better solution, but decided to simply exclude one of them...